tests/integration/restricted/validate.star

```diff
@@ -0,0 +1,27 @@
+def validate():
+    if cloned:
+       count = src_search_match_count("repo:^github.com/sourcegraph-testing/repo-ssh-keys-test$ gallery-director-drone-vacant-blizzard count:99999")
+       return count > 0
+    else:
+       return False
+
+src_create_first_admin("e2e@sourcegrah.com", "e2e-test-user", "123123123-e2e-test")
```

Comment:
I'm guessing these are utility functions provided by https://github.com/sourcegraph/src-cli/pull/200 right? Judging from the names this validation looks good

Probably a better question for https://github.com/sourcegraph/src-cli/pull/200 but what happens if any of these step fails? Do these test accounts and repos get removed?

Reply:
no, they won't get removed. it's sort of a "who's watching the watcher" situation. if they fail then i don't have much of a chance of cleaning either, but you're right: i should at least put some code in there to try

Reply:
For other cases, might be neat to provide a `defer`-style thing for cleanup purposes